### ISSUE TITLE
Exception on extension update

### ISSUE THREAD

**Issue by Pickysaurus** [Nexus-Mods/Vortex#8604]
**Describe the bug**
I have not been able to replicate this personally, but have seen several users struggle with it, so I have added it to the tracker.

When Vortex attempts to install an extension but the archive already exists in the downloads folder it throws an exception. 

The error usually looks like this:

```
dest already exists.
Error
at Object.withTmpDir (C:\Program Files\Black Tree Gaming Ltd\Vortex\resources\app.asar\renderer.js:160:210)
at C:\Program Files\Black Tree Gaming Ltd\Vortex\resources\app.asar\renderer.js:782:46
at tryCatcher (C:\Program Files\Black Tree Gaming Ltd\Vortex\resources\app.asar\node_modules\bluebird\js\release\util.js:16:23)
at Promise._settlePromiseFromHandler (C:\Program Files\Black Tree Gaming Ltd\Vortex\resources\app.asar\node_modules\bluebird\js\release\promise.js:547:31)
at Promise._settlePromise (C:\Program Files\Black Tree Gaming Ltd\Vortex\resources\app.asar\node_modules\bluebird\js\release\promise.js:604:18)
at Promise._settlePromiseCtx (C:\Program Files\Black Tree Gaming Ltd\Vortex\resources\app.asar\node_modules\bluebird\js\release\promise.js:641:10)
at _drainQueueStep (C:\Program Files\Black Tree Gaming Ltd\Vortex\resources\app.asar\node_modules\bluebird\js\release\async.js:97:12)
at _drainQueue (C:\Program Files\Black Tree Gaming Ltd\Vortex\resources\app.asar\node_modules\bluebird\js\release\async.js:86:9)
at Async._drainQueues (C:\Program Files\Black Tree Gaming Ltd\Vortex\resources\app.asar\node_modules\bluebird\js\release\async.js:102:5)
at Async.drainQueues (C:\Program Files\Black Tree Gaming Ltd\Vortex\resources\app.asar\node_modules\bluebird\js\release\async.js:15:14)
--
```

**Comment by TanninOne** [comment]
Have you checked that the user is using the latest version of vortex 1.4.2?
Because I did fix an error like this:
> exception trying to download a file that failed earlier and wasn't removed

**Comment by TanninOne** [comment]
@Pickysaurus

**Comment by Pickysaurus** [comment]
The reports I've seen are from 1.3.22 so it's entirely possible this is fixed in 1.4.x - I can't seem to get my Vortex into this situation.

**Comment by TanninOne** [comment]
Yeah, pretty sure this is fixed in 1.4.2 then but I previously thought it was new in the 1.4 beta.
